### PR TITLE
 passing air_wave into GenerateModel fun 

### PR DESCRIPTION
**Pull request by shihyuntang**
this time, not saving air_wave to self, and make it more robust with the code version of code

### CODE REVIEW

**Comment by gully** [comment]
Hey Kevin, feel free to assign me as a reviewer.

File: src/MakeModel.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ class ModelerException(Exception):
 class Modeler:
     def __init__(self, debug=False,
                  TelluricModelingDirRoot=DEFAULT_TELLURICMODELING,
-                 nmolecules=12):
+                 nmolecules=12, printoutshow=True):
```
Comment (kgullikson88) [inline src/MakeModel.py:106]: Could you rename this to something more informative, like `print_lblrtm_output`?

File: src/TelluricFitter.py [hunk 2]
```diff
@@ -68,13 +68,14 @@
 
 
 class TelluricFitter:
-    def __init__(self, debug=False, debug_level=2):
+    def __init__(self, debug=False, debug_level=2, print_show=True):
```
Comment (kgullikson88) [inline src/TelluricFitter.py:71]: This should be the same argument name as above, for consistency

File: src/TelluricFitter.py [hunk 3]
```diff
@@ -626,7 +627,8 @@ def GenerateModel(self, pars, nofit=False, separate_source=False, return_resolut
                         Ignored if nofit=False
         :param model: A DataStructures.xypoint instance containing an un-broadened telluric model.
                       If given, it uses this instead of making one.
-
+        :param air_wave: Set True of False to overwrite the air_wave saved under self. Default is None.
```
Comment (kgullikson88) [inline src/TelluricFitter.py:630]: Add to the docstring: "Default is None, which will use the `self.air_wave` attribute"

**Comment by kgullikson88** [comment]
> Hey Kevin, feel free to assign me as a reviewer.

Oi, I haven't used github in forever and don't actually know how to do that @gully

**Comment by shihyuntang** [comment]
Hi @kgullikson88 ,

ok, done! please check.

**Review by kgullikson88 — APPROVED** [review]

**Comment by kgullikson88** [comment]
Thanks, @shihyuntang!

**Pull request merged**